crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -125,6 +125,22 @@ impl UserIdentity {
         }
     }
 
+    /// Consider this identity to be trusted.
+    ///
+    /// Resets this identity to the same "pinned" state assigned the first time
+    /// a user is encountered, where we follow "trust on first use" and
+    /// consider it trusted.
+    pub async fn pin(&self) -> Result<(), CryptoStoreError> {
+        match self {
+            UserIdentity::Own(_) => {
+                // Nothing to be done for our own identity: we already
+                // consider it trusted in this sense.
+                Ok(())
```

Comment:
Right, it's more that we consider that you should always verifiy your own identity. We need to clarify this issue with own identity as it is surfacing in several places of the API
We could also error? or maybe one should do `identity.other().map(.. pin..)`?

Reply:
Do you think we should error? I think probably not but am persuadable.

Reply:
I don't know it makes me wonder if we should pin our own identity also :/ for the transition mode. Anyhow not blocking.